released/packages/coq-math-classes/coq-math-classes.8.9.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "b.a.w.spitters@gmail.com"
+
+homepage: "https://github.com/coq-community/math-classes"
+dev-repo: "git+https://github.com/coq-community/math-classes.git"
+bug-reports: "https://github.com/coq-community/math-classes/issues"
+license: "Public Domain"
+
+synopsis: ""
+description: """
+A library of abstract interfaces for mathematical structures in Coq.
```

Comment:
The description should preferably be a multi-line description of the content that expands on the synopsis. This line would fit better as synopsis. Maybe you can copy some line lines from the associated paper?

Reply:
How about this blurb from our homepage:"
Math classes is a library of abstract interfaces for mathematical structures, such as
*  Algebraic hierarchy (groups, rings, fields, …)
*  Relations, orders, …
*  Categories, functors, universal algebra, …
*  Numbers: N, Z, Q, …
*  Operations, (shift, power, abs, …)
It is heavily based on Coq’s new type classes in order to provide: structure inference, multiple inheritance/sharing, convenient algebraic manipulation (e.g. rewriting) and idiomatic use of notations."

Reply:
Thanks.